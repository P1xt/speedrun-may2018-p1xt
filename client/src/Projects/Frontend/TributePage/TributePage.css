.TributePage {
  text-align: left;
  margin-top: 40px; }
  .TributePage > h1 {
    text-transform: none; }
