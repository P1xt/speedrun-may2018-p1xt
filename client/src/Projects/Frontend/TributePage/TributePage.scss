.Tribute {
  background: #FFFFFF;
  width: 520px;
  padding: 20px;
  display: block;
  margin: 40px auto;
  box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14),
    0 1px 5px 0 rgba(0, 0, 0, 0.12);
    &__hero {
      text-align: center;
    }
    &__hero h2 {
      font-size: 1.2em;
      color: #303030;
      text-transform: uppercase;
    }
    &__content {
      max-width: 480px;
      margin: 20px auto;
    }
    &__content p {
      margin: 10px 0;
    }
}
