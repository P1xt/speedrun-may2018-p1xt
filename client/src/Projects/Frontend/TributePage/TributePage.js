import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./TributePage.css";

const TributePage = () => {
  return (
    <div className="TributePage">
      <Container>
        <Row>
          <Col>
            <h1>Tribute Page</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default TributePage;
