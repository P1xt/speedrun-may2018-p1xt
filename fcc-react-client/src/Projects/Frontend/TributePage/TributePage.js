import React from "react";

import "./TributePage.css";

const TributePage = () => {
  return (
    <div className="Tribute">
      <section className="Tribute__hero">
        <h1>Tribute to Brendan Eich</h1>
        <h2>The Creator of JavaScript</h2>
        <img
          src="/images/480px-Brendan_Eich_Mozilla_Foundation_official_photo.jpg"
          alt=""
        />
        <p>
          Photo By Darcy Padilla -{" "}
          <a
            href="https://creativecommons.org/licenses/by-sa/3.0"
            title="Creative Commons Attribution-Share Alike 3.0"
          >
            CC BY-SA 3.0
          </a>,{" "}
          <a href="https://commons.wikimedia.org/w/index.php?curid=31783773">
            Link
          </a>
        </p>
      </section>
      <section className="Tribute__content">
        <p>
          <strong>Brendan Eich</strong> (/ˈaɪk/; born July 4, 1961) is an
          American technologist and creator of the JavaScript programming
          language. He co-founded the Mozilla project, the Mozilla Foundation
          and the Mozilla Corporation, and served as the Mozilla Corporation's
          chief technical officer and briefly its chief executive officer.He is
          the CEO of Brave Software.
        </p>
        <p>
          You can read more about him on{" "}
          <a href="https://en.wikipedia.org/wiki/Brendan_Eich">Wikipedia</a>.
        </p>
      </section>
    </div>
  );
};

export default TributePage;
