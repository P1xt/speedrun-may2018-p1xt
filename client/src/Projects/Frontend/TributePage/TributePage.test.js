import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import TributePage from "./TributePage";

describe("<TributePage />", () => {
  it("renders without crashing", () => {
    shallow(<TributePage />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<TributePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
