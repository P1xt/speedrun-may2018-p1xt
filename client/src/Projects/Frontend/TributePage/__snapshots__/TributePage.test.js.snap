// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TributePage /> renders correctly 1`] = `
<div
  className="TributePage"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Tribute Page
        </h1>
      </div>
    </div>
  </div>
</div>
`;
